django/core/management/commands/makemessages.py

```diff
@@ -182,8 +182,9 @@ def write_pot_file(potfile, msgs):
         found, header_read = False, False
         for line in pot_lines:
             if not found and not header_read:
-                found = True
-                line = line.replace('charset=CHARSET', 'charset=UTF-8')
+                if line and 'charset=CHARSET' in line:
```

Comment:
The first condition is superfluous. line is a str and therefore 'charset=CHARSET' will be False if line is the empty string.